src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +137,46 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+  //get the info object from the server 
+  public ServerInfo getVersion() {
+
```

Comment:
Here and below, please remove the extra double-spacing just for consistency with the rest of the code. @lysanderp41 @mlk209